library/model_support.c

```diff
@@ -816,6 +816,10 @@ static int _cmp_map(model_map *lh, model_map *rh) {
     return rc;
 }
 
+static int null_to_json(string_buf_t *buf, int indent, int flags) {
+    return string_buf_append(buf, "null");
```

Comment:
why use "null" and not an empty json string? "{}" isn't an empty json block more like 'null'?

Reply:
oh - i see what you did from the tests now. got it

Reply:
empty object `{}` or empty string are not the same is `null`. `null` is explicit _not set_ value